salt/metalk8s/orchestrate/bootstrap_with_master.sls

```diff
@@ -31,6 +31,8 @@ Bootstrap control plane:
       - metalk8s.bootstrap.etcd
     - require:
       - salt: Bootstrap client certs
+    - pillar:
+        registry_ip: {{ pillar.get('registry_ip') }}
```

Comment:
Can't we have this merged into map.jinja or something alike so we don't have to redefine it from the orchestrate if we don't change the default value?

Reply:
we already have a ticket for that